Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/LearnWizard/POMGeneralDetailsWizardPage.xaml.cs

```diff
@@ -113,5 +120,36 @@ private void xBrowseImageLoadButton_Click(object sender, RoutedEventArgs e)
                 }
             }
         }
+
+        private void XBusinessFlowControl_ElementChangedPageEvent()
+        {            
+            if (xBusinessFlowControl != null && xBusinessFlowControl.BusinessFlow != null)
+            {
+                RepositoryItemKey repKey = new RepositoryItemKey();
+                repKey.Guid = xBusinessFlowControl.BusinessFlow.Guid;
+                repKey.ItemName = xBusinessFlowControl.BusinessFlow.Name;
+                mWizard.mPomLearnUtils.POM.MappedBusinessFlow = repKey;
+            }
+        }
+
+        private void XRDBPageURL_Checked(object sender, RoutedEventArgs e)
+        {
+            if (xPageUrlStackPanel != null && xBusinessFlowControl != null)
+            {
+                mWizard.mPomLearnUtils.POM.PageLoadFlow = ApplicationPOMModel.PageLoadFlowType.PageURL;
```

Comment:
see comment on xaml- need to merge XRDBPageURL_Checked & XRDBBF_Checked

Reply:
merged